src/components/accordion/accordion.ts

```diff
@@ -3,19 +3,21 @@
  */
 
 import { html, LitElement } from 'lit';
-import { property, customElement } from 'lit/decorators.js';
+import { property, state, customElement } from 'lit/decorators.js';
 import { AccordionItem } from './accordionItem';
 import stylesheet from './accordion.scss';
-import { unsafeSVG } from 'lit-html/directives/unsafe-svg.js';
-import onToggleSwitchSmall from '../../assets/svg/on-toggle-switch-small.svg';
 
 @customElement(`kd-accordion`)
 export class Accordion extends LitElement {
   @property({ type: Boolean }) showNumbers = false;
   @property({ type: Number }) startNumber = 1;
+  @property({ type: Boolean }) filledHeaders = false;
+  @property({ type: Boolean }) compact = false;
+  @property({ type: String }) expandLabel = 'Expand';
+  @property({ type: String }) collapseLabel = 'Collapse';

```

Comment:
Need to add JSDoc comments here so it generates the storybook props table/controls on `analyze`.